core/translate/planner.rs

```diff
@@ -97,12 +97,13 @@ fn bind_column_references(
                 return Ok(());
             }
             let mut match_result = None;
+            let normalized_id = normalize_ident(id.0.as_str());
```

Comment:
I've also found a few bugs in the previous implementation. This did not work:

```
limbo> select users.id from "users";
Parse error: Table users not found
```

because we didn't normalize the table names and the column names. Do you guys want this to go in a separate PR or can we keep it here?

Reply:
Hmm, this issue is more common:

```
limbo> select users.id from users where "id" between 1 and 10 limit 1;
Parse error: Column "id" not found
```

If we don't normalize the name in the vendored AST, we have to keep in mind it has to be normalized everywhere it's being used.

Reply:
Another one:

```
limbo> select id from users order by "id" limit 1;
Parse error: Column "id" not found
```